helmexec/exec.go

```diff
@@ -110,20 +111,22 @@ func (helm *execer) ReleaseStatus(name string) error {
 	return err
 }
 
-func (helm *execer) List(filter string) (string, error) {
+func (helm *execer) List(context HelmContext, filter string) (string, error) {
 	helm.logger.Infof("Listing releases matching %v", filter)
-	out, err := helm.exec(append([]string{"list", filter})...)
+	preArgs := context.GetPrefixArgs(helm.helmBinary)
+	out, err := helm.exec(append(preArgs, "list", filter)...)
 	helm.write(out)
 	return string(out), err
 }
 
-func (helm *execer) DecryptSecret(name string) (string, error) {
+func (helm *execer) DecryptSecret(context HelmContext, name string, flags ...string) (string, error) {
 	// Prevents https://github.com/roboll/helmfile/issues/258
 	helm.decryptionMutex.Lock()
 	defer helm.decryptionMutex.Unlock()
 
 	helm.logger.Infof("Decrypting secret %v", name)
-	out, err := helm.exec(append([]string{"secrets", "dec", name})...)
+	preArgs := context.GetPrefixArgs(helm.helmBinary)
```

Comment:
I think `GetPrefixArgs` can be more explicit for now, like `GetTillerlessArgs`.
We can generalize it to `GetPrefixArgs` once when we add other prefixes.